src/app.rs

```diff
@@ -392,6 +437,25 @@ impl<T: Send + 'static, V: View<T> + 'static> App<T, V> {
         disable_raw_mode()?;
         Ok(())
     }
+
+    #[cfg(test)]
+    pub fn get_event_tx(&self) -> tokio::sync::mpsc::Sender<Event> {
+        self.event_tx.clone()
+    }
+
+    #[cfg(test)]
+    pub fn get_terminal(&mut self) -> &mut Terminal<TestBackend> {
```

Comment:
I'd like to follow the [API guidelines of Rust](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)
```suggestion
    pub fn terminal_mut(&mut self) -> &mut Terminal<TestBackend> {
```

Reply:
Not at all. I appreciate any feedback. I misremembered the get_ prefix and wasn't aware of the _mut suffix at all.